contracts/bridge/Bridge.sol

```diff
@@ -166,45 +171,46 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable {
         }
     }
 
-    // () requests transfer KLAY to msg.sender address on relative chain.
-    function () external payable {
+    // _requestKLAYTransfer requests transfer KLAY to _to on relative chain.
+    function _requestKLAYTransfer(address _to, uint256 _fee) internal {
         require(isRunning, "stopped bridge");
         require(msg.value > 0, "zero msg.value");
+        require(msg.value > _fee, "insufficient amount");
+        require(getKLAYFee() == _fee, "invalid fee");
+
+        payKLAYFee(_fee);
 
         emit RequestValueTransfer(
             TokenKind.KLAY,
             msg.sender,
-            msg.value,
+            msg.value.sub(_fee),
             address(0),
-            msg.sender,
+            _to,
             requestNonce,
-            ""
+            "",
+            _fee
         );
         requestNonce++;
     }
 
-    // requestKLAYTransfer requests transfer KLAY to _to on relative chain.
-    function requestKLAYTransfer(address _to) external payable {
-        require(isRunning, "stopped bridge");
-        require(msg.value > 0, "zero msg.value");
+    // () requests transfer KLAY to msg.sender address on relative chain.
+    function () external payable {
+        _requestKLAYTransfer(msg.sender, getKLAYFee());
+    }
 
-        emit RequestValueTransfer(
-            TokenKind.KLAY,
-            msg.sender,
-            msg.value,
-            address(0),
-            _to,
-            requestNonce,
-            ""
-        );
-        requestNonce++;
+    // requestKLAYTransfer requests transfer KLAY to _to on relative chain.
+    function requestKLAYTransfer(address _to, uint256 _fee) external payable {
+        _requestKLAYTransfer(_to, _fee);
     }
 
     // _requestERC20Transfer requests transfer ERC20 to _to on relative chain.
-    function _requestERC20Transfer(address _contractAddress, address _from, address _to, uint256 _amount) internal {
+    function _requestERC20Transfer(address _contractAddress, address _from, address _to, uint256 _amount, uint256 _fee) internal {
         require(isRunning, "stopped bridge");
         require(_amount > 0, "zero msg.value");
         require(allowedTokens[_contractAddress] != address(0), "Not a valid token");
+        require(_fee == getERC20Fee(_contractAddress), "invalid fee");
```

Comment:
minor comment:
_fee == getERC20Fee(_contractAddress) --> getERC20Fee(_contractAddress) == _fee

Reply:
I intended to align it with `_amount > 0`. 
I wrote arguments on the left side. :)

Reply:
Then, please check _requestKLAYTransfer() on line 175.